foundations/javascript_basics/problem_solving.md

```diff
@@ -1,61 +1,68 @@
 ### Introduction
-Before we start digging into some pretty nifty JavaScript, we need to begin talking about *problem solving*: the most important skill a developer needs.
+
```

Comment:
Not sure why we have these other formatting changes.

Reply:
I had prettier activated. I reverted the changes made by prettier and made a new commit.